index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="interface">
+        <header class="header">
+            <img class="logo" src="FireBorne Entertainment.png" alt="My Logo">
+            <img class="hamburger" src="./Union.png" alt="Hamburger">
+        </header>
+        </br>
+        <h1>I'm FirePicaso. Glad to see You!</h1>
+        <p>
+            I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h2>Lets connect</h2>
+        <img class="social" src="./social media icons.png" alt="social media icons">
```

Comment:
social media icons needs to be links to your social accounts so please use e separat icon for each of them "linkedin twitter ..." wrap it in an anchor a tag and give it a valid link in the href attribute also since you have a list of them its good to organize them in a ul li tags